scripts/cypress_run.py

```diff
@@ -61,7 +63,7 @@ def run_cypress_for_test_file(
         os.environ.pop("CYPRESS_RECORD_KEY", None)
         cmd = (
             f"{XVFB_PRE_CMD} "
-            f"{cypress_cmd} --browser {browser} "
+            f"{cypress_cmd} --browser {browser} --headless "
```

Comment:
@eschutho @geido Would changing this setting to `--headless` break the screenshots used by Applitools and Cypress dashboard?

Reply:
I was testing to see if that was the case, but so far haven't seen any test failures related to it. I think it should be able to take screenshots headlessly, is my guess. This is actually the default flag, so this may already be running headlessly.